api/v1alpha2/zz_generated.deepcopy.go

```diff
@@ -22,7 +22,7 @@ package v1alpha2
 
 import (
 	"k8s.io/api/core/v1"
-	runtime "k8s.io/apimachinery/pkg/runtime"
```

Comment:
Did you regenerate this file or what led to this change?

Reply:
oops, probably my IDE gofmt/goimport-ed this file